lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -0,0 +1,122 @@
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :files_to_update,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION",
+                                       description: "Files that contain the version number and need to have it updated",
+                                       optional: false,
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :files_to_update_without_prerelease_modifiers,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION_WITHOUT_PRERELEASE_MODIFIERS",
+                                       description: "Files that contain the version number without release modifiers and need to have it updated",
+                                       optional: true,
+                                       default_value: [],
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :repo_name,
+                                       env_name: "RC_INTERNAL_REPO_NAME",
+                                       description: "Name of the repo of the SDK",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_pr_token,
+                                       env_name: "GITHUB_PULL_REQUEST_API_TOKEN",
+                                       description: "Github token to use to create the release PR",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_token,
+                                       env_name: "RC_INTERNAL_GITHUB_TOKEN",
+                                       description: "Github token to use to prepopulate the changelog",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_rate_limit,
+                                       env_name: "RC_INTERNAL_GITHUB_RATE_LIMIT_SLEEP",
+                                       description: "Sets a rate limiter for github requests when creating the changelog",
+                                       optional: true,
+                                       default_value: 0,
+                                       type: Integer),
+          FastlaneCore::ConfigItem.new(key: :branch,
+                                       description: "Allows to execute the action from the given branch",
+                                       optional: true,
+                                       default_value: "main",
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :editor,
+                                       env_name: "RC_INTERNAL_FASTLANE_EDITOR",
```

Comment:
Note that in `purchases-ios` we currently use either `FASTLANE_EDITOR` or `EDITOR` as environment variables to override this. So devs will need to update this in case they want to use their own editor.